Code/Framework/AzCore/AzCore/PlatformId/PlatformDefaults.h

```diff
@@ -50,6 +53,7 @@ namespace AZ
         AZ_ENUM_WITH_UNDERLYING_TYPE(PlatformId, int,
             (Invalid, -1),
             PC,
+            LINUX,
```

Comment:
I got to say something here:
Why not just call this `Linux` or `Linux_ID`
That would avoid all the macro issues.

Reply:
We can do the same of iOS to, which avoids the macro definition issues.

Reply:
or even 'LINUX_PLATFORM'

Reply:
Agreed, I changed it to `LINUX_ID`, I'll go ahead and change the iOS ones as well.